pkg/comments/comment.go

```diff
@@ -4,52 +4,79 @@ import (
 	"strings"
 
 	"github.com/crossplane-contrib/terrajet/pkg/markers"
+	"github.com/crossplane-contrib/terrajet/pkg/terraform/resource"
 )
 
 // Option is a comment option
 type Option func(*Comment)
 
-// WithReferenceTo returns a comment options with reference to input type
-func WithReferenceTo(t string) Option {
+// WithReferenceConfig returns a comment options with the given reference config
+func WithReferenceConfig(cfg resource.FieldReferenceConfiguration) Option {
```

Comment:
Do you think we'd need other options? I feel like in general, people would just give the whole struct for reference, setting fields in one call.